packages/nuxi/src/commands/init.ts

```diff
@@ -23,13 +23,20 @@ export default defineNuxtCommand({
       process.exit(1)
     }
 
-    const t = await downloadTemplate(template, {
-      dir: args._[0] as string,
-      force: args.force,
-      offline: args.offline,
-      preferOffline: args['prefer-offline'],
-      registry: process.env.NUXI_INIT_REGISTRY || DEFAULT_REGISTRY
-    })
+    let t
+
+    try {
+      t = await downloadTemplate(template, {
+        dir: args._[0] as string,
+        force: args.force,
+        offline: args.offline,
+        preferOffline: args['prefer-offline'],
+        registry: process.env.NUXI_INIT_REGISTRY || DEFAULT_REGISTRY
+      })
+    } catch (err) {
+      consola.error((err as Error).toString())
+      process.exit(1)
```

Comment:
Let's allow reverting to previous behaviour with a flag:

```suggestion
      if (process.env.DEBUG) {
        throw err
      }
      consola.error((err as Error).toString())
      process.exit(1)
```

@pi0 and I spoke briefly about an improvement upstream in giget to allow customising/handling different kinds of errors.